### PR TITLE
change negative arrears to positive

### PR DESCRIPTION
**Pull request by LBHABEBIC**
# What?

# Why?

# Notes

# Next steps

### CODE REVIEW

File: steps/household/rent.tsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ const CurrentBalance: React.FunctionComponent = () => {
   const tenancyUrl = process.env.TENANCY_URL;
 
   const currentBalanceValue = currentBalance.result
-    ? `£${currentBalance.result}`
+    ? `£${Math.abs(parseFloat(currentBalance.result))}`
```
Comment (erbridge) [inline steps/household/rent.tsx]: We don't want to absolute it. If it's meant to be negative, it should render with a `-`. I think you want something like:


```suggestion
    ? `${Math.sign(parseFloat(currentBalance.result)) > 0 ? "- " : ""}£${currentBalance.result}`
```

Though I think we probably want to refactor this to self-document that what we're doing here is flipping the sign.

Comment (LBHABEBIC) [reply steps/household/rent.tsx:100]: Hello @erbridge , Gill did want this to be always positive like it is in outsystems as the word arrears tells them that it's negative. do you want me to check with her?

Comment (LBHABEBIC) [reply steps/household/rent.tsx]: I spoke to Gill, I misunderstood so I will make the change as suggested.

**Comment by LBHABEBIC** [comment]
I've tried to squash the commits but having trouble with rebase -i and not sure where I'm going wrong. accidentally reverted a dependable thing.

**Review by erbridge — APPROVED** [review]

**Comment by erbridge** [comment]
Stray commit in there...

**Pull request merged**